app/src/main/res/values-de/strings.xml

```diff
@@ -549,11 +549,20 @@ Signatur: %5$s</string>
     <string name="discard_changes_question">Änderungen verwerfen?</string>
     <string name="discard">Verwerfen</string>
     <string name="keep_editing">Weiter bearbeiten</string>
+
+    <string name="channel_description_general">Nächste Vorlesung, News, Erinnerung über Studienbeiträge</string>
+    <string name="channel_description_cafeteria">Aktuelles Tagesmenü</string>
+    <string name="channel_description_chat">Neue Chat-Nachrichten</string>
+    <string name="channel_description_mvv">Abfahrten an nahegelegenen Stationen</string>
```

Comment:
'Stationen' -> 'ÖPNV Haltestellen' ?

Reply:
Went with “Haltestellen”. As this string appears below “MVV” in the Settings app, this should provide enough context.